fog/overseer/server/src/service.rs

```diff
@@ -66,15 +66,15 @@ where
         Ok(())
     }
 
-    pub fn arm(&self) -> Result<String, String> {
-        log::trace!(self.logger, "Arming overseer worker");
-        self.stop_requested.store(false, Ordering::SeqCst);
+    pub fn enable(&self) -> Result<String, String> {
+        log::trace!(self.logger, "Enabling overseer worker");
```

Comment:
Recommend `info!` so that there's a log of this happening.

Reply:
Done.